sp800-63b/sec5_authenticators.md

```diff
@@ -362,7 +362,7 @@ Biometrics SHALL be used only as part of multi-factor authentication with a phys
 
 An authenticated protected channel between sensor (or endpoint containing a sensor that resists sensor replacement) and verifier SHALL be established and the sensor or endpoint authenticated **prior** to capturing the biometric sample from the claimant.
 
-Empirical testing of the biometric system to be deployed SHALL demonstrate an EER of **1 in 1000** or better with respect to matching performance. The biometric system SHALL operate with an FMR of **1 in 1000** or better.
+The biometric system SHALL operate with a false match rate (FMR) of **1 in 1000** or better. The false non-match rate (FNMR) SHOULD be low enough that the claimant has a good chance of authenticating successfully within the consecutive failed authentication attempt limits listed below.
```

Comment:
I think we drop the second sentence. We set the security requirement, let the RP set the usability requirement.  As it stands, this requirement (or optional requirement) is not testable.